juniper_official/vpn/l2vpn/check-l2circuit-pw-state.rule

```diff
@@ -75,22 +97,94 @@
+                                        release-support min-supported-release;
+                                    }
+                                }
+                                platforms MX204 {
+                                    releases 23.2 {
+                                        sensors isis-oc-junos;
+                                        release-support min-supported-release;
+                                    }
+                                    releases 22.2 {
+                                        sensors isis-oc;
                                         release-support min-supported-release;
                                     }
                                 }
                             }
-                         }
-                         operating-system junosEvolved {
+                        }
+                        operating-system junosEvolved {
                             products ACX {
-                                platforms All {
-                                    releases 22.1R1 {
+                                sensors isis-oc;
+                                platforms ACX7024 {
+                                    releases 23.2R1 {
+                                        sensors isis-oc-junos;
+                                        release-support min-supported-release;
+                                    }
+                                    releases 22.3R1 {
+                                        sensors isis-oc;
+                                        release-support min-supported-release;
+                                    }
+                                }
+                                platforms ACX7100 {
+                                    releases 23.2R1 {
+                                        sensors isis-oc-junos;
+                                        release-support min-supported-release;
+                                    }
+                                    releases 22.3R1 {
+                                        sensors isis-oc;
+                                        release-support min-supported-release;
+                                    }
+                                }
+                                platforms ACX7348 {
+                                    releases 23.2R1 {
+                                        sensors isis-oc-junos;
+                                        release-support min-supported-release;
+                                    }
+                                    releases 22.3R1 {
+                                        sensors isis-oc;
+                                        release-support min-supported-release;
+                                    }
+                                }
+                                platforms ACX7024X {
+                                    releases 23.2R1 {
+                                        sensors isis-oc-junos;
```

Comment:
Change the sensor name to actual "l2ckt"

Reply:
updated with l2ckt